File: 1. GreetingApp/app/src/main/java/com/mo/greetingapp/MainActivity.java
package com.mo.greetingapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {


    private EditText editText;
    private Button button;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        editText = findViewById(R.id.editText);
        button = findViewById(R.id.button);


        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                String enteredText  = editText.getText().toString();
                Toast.makeText(MainActivity.this, "Welcome, "+enteredText, Toast.LENGTH_SHORT).show();
            }
        });







    }
}

File: __Kotlin Apps/Greeting App/app/src/main/java/com/example/greetingapp/MainActivity.kt
package com.example.greetingapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.Toast


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val et :EditText = findViewById(R.id.editTextText)
        val button :Button = findViewById(R.id.button)


        button.setOnClickListener(){
            val etText : String = et.text.toString()
            
            Toast.makeText(this, "Hello $etText",Toast.LENGTH_SHORT).show()

        }


    }
}